packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue

```diff
@@ -27,6 +29,23 @@ const emit = defineEmits(['open-drilldown', 'append-input-port', 'update-state']
 const modelInput = props.node.inputs.find((input) => input.type === 'modelId');
 const isModelInputConnected = computed(() => modelInput?.status === WorkflowPortStatus.CONNECTED);
 
+watch(
+	() => modelInput?.value,
+	async () => {
+		if (modelInput?.value) {
+			const modelId = modelInput.value[0];
+			const modelConfigurations = await getModelConfigurations(modelId);
```

Comment:
As something sort of unrelated we may want to rename this function - i just did a deep dive in the code to look at `getModelConfigurations` because I assumed it was a model config endpoint call. Perhaps something like `getModelConfigurationsForModel` or something?

Reply:
ya I can definitely rename